02.Rmd

```diff
@@ -0,0 +1,24 @@
+# Statistical Learning
+
+**Learning objectives:**
+
+- Define Statistical Learning.
```

Comment:
As we talked about on Slack, this should be a little broader. "Describe how statistical learning can be used to understand data" or something, maybe?

Reply:
Oh, it's also possible this LO just goes away, if the other ones cover all the expansion.

Reply:
yeah, the expansion feels a little gratuitous, so I'll just remove it.